libcxx/include/__exception/exception.h

```diff
@@ -22,8 +22,12 @@
 #endif
 
 #ifdef STD_EXCEPTION_HAS_STACK_TRACE
+#if defined(OS_DARWIN)
```

Comment:
Please add a comment here, saying that this is a ClickHouse-specific patch of LLVM + the motivation + (if possible) a reference to an issue.